setup.py

```diff
@@ -27,6 +24,7 @@
     'pylint',
     'pytest',
     'pytest-watch',
+    'osmosis-on-premise-driver',
```

Comment:
I find it strange that the `osmosis-on-premise-driver` Python package is needed to run the unit tests but it's not needed during the course of normal usage.

Reply:
This is because this create a circular dependency, I only added in the tests to be sure that the load of one of the drivers was working good.